### PR TITLE
n-message hover will not destroyed (#1036)

### LINKED ISSUE #1036: Message组件鼠标悬停不销毁（Message component is not destroyed when the mouse hover）
### This function solves the problem (这个功能解决的问题)
鼠标悬停在message信息组件上时，此信息永不消失，鼠标移开了间隔一定时间才消失。使用场景：当提示信息过长时，用户可复制此提示信息。When you hover over a message component, the message never disappears until the mouse moves away for a certain amount of time.  Usage scenario: When the message is too long, you can copy the message
### Expected API (期望的 API)
鼠标悬停，当前提示信息实例不销毁
The current prompt instance is not destroyed when the mouse hover over it

### PR DESCRIPTION
**Pull request by Volankey**
close #1036

### CODE REVIEW

**Review by 07akioni — COMMENTED** [review]
1. This should be a optional feature. Add `keepAliveOnHover` on `n-message-provider` and message option.
2. Duration shouldn't be recaculate after mouse leave.

**Comment by Volankey** [comment]
> 1. This should be a optional feature. Add `keepAliveOnHover` on `n-message-provider` and message option.
> 2. Duration shouldn't be recaculate after mouse leave.



> 1. This should be a optional feature. Add `keepAliveOnHover` on `n-message-provider` and message option.
> 2. Duration shouldn't be recaculate after mouse leave.

I don’t agree with the second point. I refer to` elementui` and `antd`, i think it should be recaculate.

**Comment by 07akioni** [comment]
> > 1. This should be a optional feature. Add `keepAliveOnHover` on `n-message-provider` and message option.
> > 2. Duration shouldn't be recaculate after mouse leave.
> 
> > 1. This should be a optional feature. Add `keepAliveOnHover` on `n-message-provider` and message option.
> > 2. Duration shouldn't be recaculate after mouse leave.
> 
> I don’t agree with the second point. I refer to` elementui` and `antd`, i think it should be recaculate.

Okay, just implement the firsrt

File: src/message/demos/zhCN/basic.demo.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ```html
 <n-space>
-  <n-button @click="info"> 信息 </n-button>
+  <n-button @click="info"> 信息（Hover不消失）</n-button>
```
Comment (07akioni) [inline src/message/demos/zhCN/basic.demo.md:5]:
```suggestion
  <n-button @click="info"> 信息（Hover 不消失）</n-button>
```

File: src/message/src/Message.tsx [hunk 2]
```diff
@@ -123,6 +123,8 @@ export default defineComponent({
     return (
       <div
         class={`${mergedClsPrefix}-message-wrapper`}
+        onMouseenter={this.onMouseenter}
+        onMouseleave={this.onMouseleave}
```
Comment (07akioni) [inline src/message/src/Message.tsx:127]: 这事件会冒泡的，事件处理器要注意这种情况

**Pull request merged**